heat/spatial/distance.py

```diff
@@ -0,0 +1,317 @@
+import torch
+import numpy as np
+from mpi4py import MPI
+
+from ..core import factories
+from ..core import types
+
+__all__ = ["cdist", "rbf"]
+
+
+def _euclidian(x, y):
```

Comment:
need some short docs for the internal functions and proper docs for the functions which are visible from outside (cdist and rbf)